src/main/scala/inox/ast/TypeOps.scala

```diff
@@ -174,6 +174,8 @@ trait TypeOps {
         TupleType(ts).unveilUntyped
       }
 
+    // TODO BVType, maybe?
```

Comment:
Hmm, I'm not sure. I think I'd prefer them to be disjoint in Inox. In Stainless we can maybe add auto-casting.